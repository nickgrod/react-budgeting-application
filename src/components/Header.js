import React from 'react'
import {NavLink} from 'react-router-dom'

const Header = () => (
    <header>
        <h1>Budget App</h1>
        <NavLink exact to = '/' activeClassName="onPage">Go Home</NavLink>
        <NavLink exact to = '/create' activeClassName="onPage">Create Page</NavLink>
    </header>
)
export default Header