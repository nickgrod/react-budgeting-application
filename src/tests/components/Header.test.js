import React from 'react'
import { shallow } from 'enzyme'
import Header from '../../components/Header'

test('Should render the header correctly', () => {
    const wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot()
})