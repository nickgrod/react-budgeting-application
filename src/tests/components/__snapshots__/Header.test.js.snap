// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the header correctly 1`] = `
<header>
  <h1>
    Budget App
  </h1>
  <NavLink
    activeClassName="onPage"
    exact={true}
    to="/"
  >
    Go Home
  </NavLink>
  <NavLink
    activeClassName="onPage"
    exact={true}
    to="/create"
  >
    Create Page
  </NavLink>
</header>
`;
